osu.Game/Screens/Play/SkipOverlay.cs

```diff
@@ -43,8 +48,7 @@ public partial class SkipOverlay : CompositeDrawable, IKeyBindingHandler<GlobalA
         private FadeContainer fadeContainer;
         private double displayTime;
 
-        private bool isClickable;
-        private bool skipQueued;
+        private bool stillClickable = true;
```

Comment:
I'm not entirely sure why you set this to `true`, but the source of the problem is that this should only become true after gameplay has started and clock is running.